preferences/Preference.js

```diff
@@ -1,19 +1,23 @@
 // See https://github.com/jenkinsci/js-storage
-const storage = require('@jenkins-cd/storage');
-const jenkinsNS = storage.jenkinsNamespace();
-const preferences = jenkinsNS.subspace('preferences');
+const storage = require('@jenkins-cd/storage/storage');
+const StorageNamespace = require('@jenkins-cd/storage/StorageNamespace');
```

Comment:
Unless `StorageNamespace` is holding some sort of state (and I'm 99.9% sure it doesn't) that needs to be global (in the page), then this will be okay. Yes it might end up bundling the module where it's not ideal to do so, but not the end of the world. We'll be fixing that issue (some time) and so it should just go away.